docs/arcaflow/workflows/flow-control.md

```diff
@@ -2,6 +2,258 @@
+
+##### How to do this in a workflow
+
+There are two tags that create the described OR dependency.
+
+| Tag           | Description                                      |
+|---------------|--------------------------------------------------|
+| `!oneof`      | A general use tag to depend on any of the inputs |
+| `!ordisabled` | A special case of `!oneof` that automatically handles the disabled case |
+
+##### How to use `!oneof`
+
+To use `!oneof` for graceful handling of disabled steps, the oneof should depend on either ONE OF the success output or the disabled output.
+
+The oneof tag is a method of creating a schema `one_of_string` type from values present in the workflow.
+
+The syntax of `!oneof` is:
+- Following the tag `!oneof`, create a new YAML section (map) by moving to a new line. That section should contain two properties:
+  - discriminator: A string that specifies what the oneof discriminator should be.
+  - options: A YAML section (map) that contains all options. The keys are the discriminator values, and the values should be valid expressions.
+
+Example:
+```yaml title="workflow.yaml"
+version: v0.2.0
+input:
+  root: RootObject
+  objects:
+    RootObject:
+      id: RootObject
+      properties:
+        step_1_enabled:
+          type:
+            type_id: bool
+        step_1_input:
+          type:
+            type_id: integer
+steps:
+  my_step:
+    plugin:
+      deployment_type: image
+      src: path/to/my_plugin:1
+    input:
+      param_1: !expr $.input.step_1_input
+    enabled: !expr $.input.step_1_enabled
+outputs:
+  workflow_success:
+    plugin_output: !oneof
+      discriminator: result
+      options:
+        enabled: !expr $.steps.my_step.outputs.success
+        disabled: !expr $.steps.my_step.disabled.output
+```
+
+##### How to use `!ordisabled`
+
+Since arcaflow needed a simple way to disable a step easily, there is a method that is as simple as changing the expression tag from `!expr` to `!ordisabled`
+
+Using `!ordisabled`, the following example is otherwise equivalent to the prior example:
+```yaml title="workflow.yaml"
+version: v0.2.0
+input:
+  root: RootObject
+  objects:
+    RootObject:
+      id: RootObject
+      properties:
+        step_1_enabled:
+          type:
+            type_id: bool
+        step_1_input:
+          type:
+            type_id: integer
+steps:
+  my_step:
+    plugin:
+      deployment_type: image
+      src: path/to/my_plugin:1
+    input:
+      param_1: !expr $.input.step_1_input
+    enabled: !expr $.input.step_1_enabled
+outputs:
+  workflow_success:
+    plugin_output: !ordisabled $.steps.my_step.outputs.success
+```
+
+#### Alternative methods
+
+For handling disabled steps, `!oneof` and `!ordisabled` are the recommended methods because they cause output failure when the step fails. However, if it is acceptable for the workflow to succeed when a step crashes, the optional tags could be the right feature for your workflow.
```

Comment:
Maybe "optional" in "optional tags" should be emphasized or quoted. We're talking about the "optional" family of expression tags, not just any generic tags that happen to be optional.

Reply:
I re-worded it.